_data/authors.yml

```diff
@@ -0,0 +1,9 @@
+timo_daum:
```

Comment:
maybe prepare already some default TWC author?

Reply:
Agreed. It would be TechWorkersBER then, what should "name" be? "Berlin Tech Workers Coalition?"

Reply:
yes, it's fine